server.js

```diff
@@ -32,7 +38,7 @@ app.get("/", (request, response) => {
     request.session.views = 1;
     console.log("[New-visit] %s", request.session.id);
   }
-  response.sendFile(__dirname + "/build/index.html");
+  response.json({ session: request.session.id });
```

Comment:
this could have been `response.end()`
but you might want to keep it for now for dev purposes? so that the browser can console.log the session id easily